version.go

```diff
@@ -810,3 +810,23 @@ func hidePair(pvl []PairedVersion) []Version {
 	}
 	return vl
 }
+
+// Decompose a Version into the underlying number, branch and revision
```

Comment:
Let's rename this to something a little more descriptive, if we're gonna export it - `Info` is pretty bland 😄  

how about `VersionComponentStrings()`?